cypress/objects/taskPage.js

```diff
@@ -11,7 +11,7 @@ class Tasks {
     getFieldTextArea() {
```

Comment:
Just like with "small field text" this is not descriptive, should probably be changed to something like "getTaskDescriptionTextArea", agree? You can fix this in this pull-request or some other, but otherwise I'm giving this one a go, amazing job :) 👍